layout/header/header-menu/component.jsx

```diff
@@ -40,7 +40,7 @@ const HeaderMenu = () => {
 
           const activeClassName = classnames({ '-active': item.pages && item.pages.includes(pathname) });
           let DropdownMenu;
-          if (item.id !== 'blog') {
+          if (item.id !== 'blog' && item.id !== 'azavea-test') {
```

Comment:
This feels like it's in the wrong place. I would think it would be better as a property in the `APP_HEADER_ITEMS` constant, e.g. `menu: false`.  But refactoring that isn't something to deal with in this test page issue.

Reply:
Yeah I'm completely in agreement, this was a bit of a surprise to me. My best guess as to the way things are evolving is that we are more likely to add an item to the `Dashboards` dropdown than to make a whole new header link, so I suspect this will likely be moot, but if it turns out that we do need to do this then I agree that a refactor would be worthwhile.